src/data_gradients/feature_extractors/segmentation/classes_heatmap_per_class.py

```diff
@@ -43,11 +43,13 @@ def description(self) -> str:
             "The heatmap represents areas of high object density within the images, providing insights into the spatial distribution of objects. "
             "By examining the heatmap, you can quickly detect whether objects are predominantly concentrated in specific regions or if they are evenly "
             "distributed throughout the scene. This information can serve as a heuristic to assess if the objects are positioned appropriately "
-            "within the expected areas of interest."
+            "within the expected areas of interest.<br/>"
+            "Note that images are resized to a square of the same dimension, which can affects the aspect ratio of objects. "
```

Comment:
affect*

Reply:
fixed in 155ecb7b5da7a633c2ebadda815601cadff1f4aa